setup.py

```diff
@@ -68,6 +69,7 @@
         'pandas', 'lxml', 'uncertainties'
     ],
     'extras_require': {
+        'depletion-mpi': ['mpi4py'],
```

Comment:
We have something like this in the [User's guide - prereqs](https://docs.openmc.org/en/stable/usersguide/install.html?highlight=mpi4py#installing-in-development-mode). Do we want to include additional instructions for installing via `python setup.py install -e .[depletion-mpi]`, that would pull these optional dependencies as well?

Reply:
I'm actually working on another branch to allow depletion with MPI without necessarily having parallel HDF5 (which can be a pain to install on some systems). I'll update the user guidance in that branch if it's ok with you.

Reply:
Very nice. Yeah, that's good with me